types/v56/vm_types.go

```diff
@@ -0,0 +1,39 @@
+package types
```

Comment:
Why a separate file for some VM related types?
If we split by entity, this file should contain all VM types, but then we should have separate files for each entity

Reply:
I am trying to start splitting by domain(VM/VAPP/EdgeGtw/ etc.). If everyone agrees then it is start to add new one to separate `type` files. In time we move everything accordingly. I won't do let's split everything now it's to much and not this PR scope, but we can start doing better from today.